src/main/java/mops/portfolios/Group.java

```diff
@@ -0,0 +1,50 @@
+package mops.portfolios;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.json.JSONException;
+import org.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+
+
+@Service
+public class Group {
+
+  private static final Logger logger = LoggerFactory.getLogger(PortfoliosApplication.class);
+
+  Group get(int id) {
+    return this;
+  } //TODO
+
+  /**
+   * Use this method to get the updates from Gruppenbildung regarding groups.
+   * @param matrikelnr The MatrikelNr of the student to get the group members of
+   * @return List of the MatrikelNr of each group member
+   */
+
+  List<MatrikelNr> getGroupmembers(MatrikelNr matrikelnr) {
+    HttpClient httpClient = new HttpClient();
+    // TODO genaues URI mit gruppen2 absprechen
+    String responseBody = httpClient.get("/gruppen2/groupmembers");
+
+    if (responseBody == null) {
+      // TODO: use data from database further
+    } else {
+      JSONObject jsonObject;
```

Comment:
PMD fails here as jsonObject not used yet

This will be fixed later as it will be used for the data received by gruppen2